packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -1381,11 +1379,11 @@ class CheckboxMenuButton extends StatelessWidget {
       onPressed: onChanged == null ? null : () {
         switch (value) {
           case false:
-            onChanged!.call(true);
+            onChanged!(true);
```

Comment:
I assume these were originally the result of a dart fix or something...

Reply:
Dart is sort of inconsistent with regards to null checks: `!` works fine without an extra `.call()`, but `?` does not. I'm guessing that's why the original was set up the way it was.

I filed https://github.com/dart-lang/language/issues/3685 a while back to see if we could make it more consistent, but it looks like it probably won't change.